api/ingest-tokens.go

```diff
@@ -104,7 +104,7 @@ func (i *IngestTokens) Add(repositoryName string, tokenName string, parserName s
 func (i *IngestTokens) Update(repositoryName string, tokenName string, parserName string) (*IngestToken, error) {
 	var mutation struct {
 		Result struct {
-			IngestToken ingestTokenData
+			Repository Repository
```

Comment:
Given that we don't even use the Repository here, we may just do the same as for the Remove method, namely just query for
```
Type string `graphql:"__typename"`
```